release/karaf/distro/README_FUSE_INTEGRATION.md

```diff
@@ -11,7 +11,7 @@ Camel components used in kie: kie-camel and jbpm-workitem-camel components
 
 Requirements
 ============
-DROOLS_VERSION --> 6.2.0.Final
+DROOLS_VERSION --> ${DROOLS_VERSION}
```

Comment:
Hi Petr,

We agree - I have amended this and pushed the change based on your feedback.

Best,

Ant

On 27/11/15 09:38, Petr Široký wrote:
>
> In release/karaf/distro/README_FUSE_INTEGRATION.md 
> <https://github.com/jboss-integration/fuse-bxms-integ/pull/172#discussion_r46030489>:
>
> > @@ -11,7 +11,7 @@ Camel components used in kie: kie-camel and jbpm-workitem-camel components
> >
> >  Requirements
> >  ============
> > -DROOLS_VERSION --> 6.2.0.Final
> > +DROOLS_VERSION --> ${DROOLS_VERSION}
>
> This should be I believe removed or changed to something like 
> |DROOLS_VERSION => 6.3.0.Final|. The current form does not add any 
> additional info.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/jboss-integration/fuse-bxms-integ/pull/172/files#r46030489>.
>

-- 
Ant Stephenson
Red Hat UK